Documentation/event-reference.md

```diff
@@ -0,0 +1,204 @@
+
+#### Data payload definition<!-- omit in toc -->
+
+This event has no data payload.
+
+-----
+
+## Events dispatched by Edge
+
+The following events are dispatched by the Edge extension client-side.
+
+### Edge identity response
+
+This event is a response to the [Edge request identity event](#edge-request-identity)) and provides the location hint being used by the Edge Network extension in requests to the Edge Network. The Edge Network location hint may be used when building the URL for Edge Network requests to hint at the server cluster to use.
+
+| Value | Behavior |
+| ------ | -------- |
+| Yes (`y`) | Hits are sent |
+| No (`n`) | Hits are dropped and not sent |
+| Pending (`p`) | Hits are queued until `y`/`n` is set; when set, queued events follow the value's behavior |
+
+#### Event Details<!-- omit in toc -->
+
+| Event type                        | Event source                          |
+| --------------------------------- | ------------------------------------- |
+| com.adobe.eventType.edge          | com.adobe.eventSource.responseIdentity |
+
+#### Data payload definition<!-- omit in toc -->
+
+| Key       | Value type    | Required | Description           |
+| --------- | ------------- | -------- | --------------------- |
+| locationHint  | `String` | Yes       | The Edge Network location hint currently set for use when connecting to Edge Network. |
+
+----- 
+
+### Edge content response error
+
+This event is an error response to an originating event. If there are multiple error responses for a given triggering event, separate error event instances will be dispatched for each error. 
+
+#### Event Details<!-- omit in toc -->
+
+| Event type                        | Event source                          |
+| --------------------------------- | ------------------------------------- |
+| com.adobe.eventType.edge          | com.adobe.eventSource.errorResponseContent |
+
+#### Data payload definition<!-- omit in toc -->
+
+| Key       | Value type    | Required | Description           |
+| --------- | ------------- | -------- | --------------------- |
+| requestId  | `String` | Yes       | The ID (`UUID`) of the batched Edge Network request tied to the event that triggered the error response. |
+| requestEventId  | `String` | Yes       | The ID (`UUID`) of the event that triggered the error response. |
+
+----- 
+
+### Edge event response
+
+This event is a response to an event.
+
+
+#### Event Details<!-- omit in toc -->
+
+| Event type                        | Event source                          |
+| --------------------------------- | ------------------------------------- |
+| com.adobe.eventType.edge          | This value is copied from the event being responded to; if not set or `null`, has a default value of com.adobe.eventSource.responseContent |
```

Comment:
just `com.adobe.eventSource.responseContent`

Reply:
We should have error response and responseContent. I am not sure how to accommodate any source. @emdobrin @kevinlind Do we have any a limited number eventSource that we expect or this can be anything and we should not try to document each and every known source?